### DISCUSSION TITLE
Villager trading on multiplayer servers

### DISCUSSION THREAD

**Discussion by Mernst0318** [PurpurMC/Purpur#746] [__Dismissed__] [answered]
A option that allows villager's discounted prices to be fluid throughout the servers player base would be useful.  This would be purely for QOL.

I.E. An option that would allow Player A to cure a zombie villager multiple times and receive the discount as well as Player B being able to trade with the same villager at the discounted price without having to cure the villager himself.  

In my case, this is very useful as a few friends and myself are building a large villager trading hall on a server with 3-4 people and being able to take turns curing the villagers would save significant time and would more easily allow for us to build a large trading hall instead of each of us having small separate trading halls.  

The other benefit to this is due to the option to fix zombie villager trading (fix-curing-zombie-villager-discount-exploit: false) seems to not always work 100% of the time and reset the prices for some players on the server.

Thank you for your time,
Mernst0318

**Comment by encode42** [comment, marked as answer]
Duplicate of #680
